manager/integration/tests/common.py

```diff
@@ -1684,6 +1684,33 @@ def wait_for_volume_migration_node(client, volume_name, node_id):
     return v
 
 
+def wait_for_volume_snapshot_purge_completion(client, volume_name):
```

Comment:
Feels like we should add those into `wait_for_snapshot_purge`. Validate it using both progress and snapshot.